src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -98,7 +99,30 @@ def process(self, *args, **kwargs):
         pass
 
 
-class NativeAppManager(SqlExecutionMixin):
+class FindObjectRowMixin:
```

Comment:
What is the benefit to making a new Mixin as opposed to introducing the main method `find_row_by_object_name` in `SqlExecutionMixin` ?  New managers would only need to inherit from the`SqlExecutionMixin`, and we do not need to remember this additional class. It is possible that those who forget to include this new class when defining a new manager end up creating their own such method, which could be avoided if it were already contained in `SqlExecutionMixin`

Reply:
FWIW, I think that if this can be included in SqlExecutionMixin then it would be most useful in there. I just wasn't initially sure whether SqlExecutionMixin is meant to be, as Cam says, "lower-level than these sort of helpers."

Reply:
I see your point @sfc-gh-cgorrie and it makes sense. Will leave it up to you, happy to approve once you have decided however you want to proceed.